tests/tests_unit/tests_db_connection_unit/test_kafka_unit.py

```diff
@@ -0,0 +1,243 @@
+import os
+import re
+
+import pytest
+
+from onetl.connection import Kafka
+
+pytestmark = pytest.mark.kafka
+
+
+@pytest.mark.parametrize(
+    "scala_version,spark_version",
+    [
+        ("2.11", "2.3.0"),
+        (None, "2.3.0"),
+        ("2.12", "3.3.0"),
+        (None, "3.3.0"),
+    ],
```

Comment:
```suggestion
    "spark_version, scala_version_input,scala_version_real",
    [
        ("2.3.0", None, "2.11"),
        ("2.3.0", "2.11", "2.11"),
        ("2.3.0", "2.12", "2.12"),
        ("3.3.0", None, "2.12"),
        ("3.3.0", "2.12", "2.12"),
        ("3.3.0", "2.13", "2.13"),
    ],
```
Использовать разные переменные в аргументе функции и в проверке ее результата, if из теста убрать